docs/getting-started/freqaskques.md

```diff
@@ -97,7 +97,21 @@ To get up and running with the Zowe CLI component quickly, see [Zowe CLI quick s
 
 <summary></summary>
 
-The primary prerequisites is Java on z/OS and the z/OS Management Facility enabled and configured. For a complete list of software requirements listed by component, see [System requirements](../user-guide/systemrequirements.md).
+Prerequisites vary by component used, but in most cases the primary prerequisites are Java and NodeJS on z/OS and the z/OS Management Facility enabled and configured. For a complete list of software requirements listed by component, see [System requirements](../user-guide/systemrequirements.md).
```

Comment:
Users must have Java and NodeJS enabled and configured on z/OS and z/OS management facility. 
**Note**: Prerequisites can vary based on what components are being used.

Reply:
It's not a "must" if Docker is used, that's why I changed it.